common/supplemental/supplementalData.xml

```diff
@@ -2783,7 +2783,8 @@ XXX Code for transations where no currency is involved
 		<territory type="CN" gdp="25360000000000" literacyPercent="95.1" population="1394020000">	<!--China-->
 			<languagePopulation type="zh" populationPercent="90" officialStatus="official"/>	<!--Chinese-->
 			<languagePopulation type="wuu" populationPercent="6"/>	<!--Wu Chinese-->
-			<languagePopulation type="yue_Hans" populationPercent="5.2" references="R1219"/>	<!--Cantonese (Simplified)-->
+			<languagePopulation type="yue" populationPercent="5.2" references="R1219"/>	<!--Cantonese (Simplified)-->
+			<languagePopulation type="yue_Hant" populationPercent="5.2" references="R1219"/>	<!--Cantonese (Traditional)-->
```

Comment:
This is wrong. `Hant` is the default for `yue` so `yue` by itself is the same thing as `yue_Hant`. We already had `yue_Hans` and need to keep that (the non-default script), we just needed to add `yue` by itself.

Reply:
right, that's what I had in [In supplementalData, you can check the languageData, if there is only one non-secondary script for the language (does not apply to yue), that is presumably the primary.](https://github.com/unicode-org/cldr/pull/3945#pullrequestreview-2226108995) [just different order]

Reply:
Okay, I will fix it. I wasn't sure how the script defaults are handled in this file vs elsewhere. Thank you for answering all my questions :)